src/plots/gl2d/scene2d.js

```diff
@@ -286,10 +292,11 @@ proto.updateFx = function(options) {
 
     fullLayout.dragmode = options.dragmode;
     fullLayout.hovermode = options.hovermode;
-};
 
-var relayoutCallback = function(scene) {
+    this.graphDiv._fullLayout = fullLayout;
```

Comment:
That's an idea.

So, the `_fullLayout`  gets lost in `Plots.supplyDefaults`. We could add a block handling refs on scene objects similar to this [one](https://github.com/plotly/plotly.js/blob/9b7c18f5f576f103b65889574d25ad9924d4d7c1/src/plots/plots.js#L494-L499) for cartesian axes. But I think we could do better.

The reason the lost of reference only affects `dragmode` relayouts is because all other relayout calls pass through `scene2d.plot` which update the `fullLayout` ref [here](https://github.com/plotly/plotly.js/blob/9b7c18f5f576f103b65889574d25ad9924d4d7c1/src/plots/gl2d/scene2d.js#L353) on every pass.